ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/capabilities/provision/cucushift-installer-rehearse-azure-ipi-capabilities-provision-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-capabilities-provision
```

Comment:
How about rename it to `cucushift-installer-rehearse-azure-ipi-capability-addtionalcaps-provision`?
1. the change of `capabilities` is to keep consistent with `cucushift-installer-rehearse-gcp-ipi-capability-addtionalcaps-provision`
2. the addon for `addtionalcaps` is to indicate the install workflow enabled addtional cap coverage.